torch/_dynamo/_trace_wrapped_higher_order_op.py

```diff
@@ -0,0 +1,109 @@
+import functools
+
+import torch.utils._pytree as pytree
+from torch._C import DispatchKey
+from torch._higher_order_ops.utils import autograd_not_implemented
+
+from torch._ops import HigherOrderOperator
+from torch._subclasses import FakeTensorMode
+
+from torch.fx.experimental.proxy_tensor import ProxyTorchDispatchMode, track_tensor_tree
+from torch.utils._python_dispatch import _get_current_dispatch_mode
```

Comment:
can you add an `__all__` to this file to make it clearer what the public API is

Reply:
yea